### PR TITLE
add paperjs Raster type

### PR DESCRIPTION
**Pull request by anotherjesse**
Sample:

      <Group>
        <Raster
          source="http://assets.paperjs.org/images/marilyn.jpg"
          position={[100, 100]}
        />
        <Circle center={[80, 80]} radius={35} clipMask={true} />
      </Group>

### CODE REVIEW

**Comment by anotherjesse** [comment]
@psychobolt I am new to react/js - so I'm not certain if all the places I added `Raster` were required.

For instance I saw that `Rectangle` is not included in `export type Paper = {`... Is that by design or an accident?

**Comment by psychobolt** [comment]
It is not by design that I didn't implement Rectangle type. As of now flow types are not fully implemented.

File: src/Paper.types.js [hunk 1]
```diff
@@ -45,6 +47,7 @@ const TYPES: Types = {
   [CONSTANTS.Layer]: (props, paper) => new paper.Layer(props),
   [CONSTANTS.Group]: (props, paper) => new paper.Group(props),
   [CONSTANTS.Path]: (props, paper) => new paper.Path(props),
+  [CONSTANTS.Raster]: (props, paper) => new paper.Raster(props),
```
Comment (psychobolt) [inline src/Paper.types.js]: Constructor takes a certain parameters: http://paperjs.org/reference/raster/#. You need to map the prop object to the correct constructor usage.

Comment (psychobolt) [reply src/Paper.types.js:50]: My mistake, I see that Paper supports size as a option.

**Pull request merged**